internal/cli/atlas/backup/backup.go

```diff
@@ -51,6 +52,7 @@ func AtlasCLIBuilder() *cobra.Command {
 		snapshots.Builder(),
 		restores.AtlasCLIBuilder(),
 		exports.Builder(),
+		schedule.AtlasCLIBuilder(),
```

Comment:
we normally use builder, only if there are differences between AtlasCLI and MongoCLI we might export different methods.

```suggestion
		schedule.Builder(),
```

Reply:
I had some issues with this builder previously, I don't really remember why, that's why I named it `AtlasCLIBuilder`. I just renamed it to `Builder` and it still works fine, so that's good.